cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/DefaultNetworkConnector.java

```diff
@@ -17,14 +17,14 @@
 import com.sequenceiq.cloudbreak.cloud.model.SubnetSelectionParameters;
 import com.sequenceiq.cloudbreak.cloud.model.SubnetSelectionResult;
 
-public abstract class DefaultNetworkConnector implements NetworkConnector {
+public interface DefaultNetworkConnector extends NetworkConnector {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultNetworkConnector.class);
+    Logger LOGGER = LoggerFactory.getLogger(DefaultNetworkConnector.class);
 
-    private static final String NOT_ENOUGH_AZ = "Acceptable subnets are in %d different AZs, but subnets in %d different AZs required.";
+    String NOT_ENOUGH_AZ = "Acceptable subnets are in %d different AZs, but subnets in %d different AZs required.";
```

Comment:
Not pressing, but something we have to look out for. GCP subnets span across all the AZs in that region, and the user can not control it. So giving a subnet is enough in GCP.